Makefile

```diff
@@ -2,20 +2,23 @@ PKGS=$(shell go list ./... | grep -v /vendor)
 SHELL_IMAGE=golang:1.8.3
 GIT_SHA=$(shell git rev-parse --verify HEAD)
 VERSION=$(shell cat VERSION)
+PWD=$(shell pwd)
 
 default: bindata compile
+
 compile:
-	@go build -o ${GOPATH}/bin/kubicorn -ldflags "-X github.com/kris-nova/kubicorn/cmd.GitSha=${GIT_SHA} -X github.com/kris-nova/kubicorn/cmd.Version=${VERSION}" main.go
+	go build -o bin/kubicorn -ldflags "-X github.com/kris-nova/kubicorn/cmd.GitSha=${GIT_SHA} -X github.com/kris-nova/kubicorn/cmd.Version=${VERSION}" main.go
```

Comment:
Not annoying - I like this plan.

1. `make` compile only into `./bin`
2. `make install` move compiled files from `./bin` -> `$GOPATH/bin` (we should probably use `go install here`)
3. `make all` wraps the above two targets